docs/source/changelog.rst

```diff
@@ -6,13 +6,15 @@ Changelog
 
 - Resolve issue where the optional ``--noauth_local_webserver`` command line argument would not be propagated during the authentication process. (:issue:`35`)
 - Drop support for Python 3.4 (:issue:`40`)
+- When using ```to_gbq``` if ```if_exists``` is set to ```append```, dataframe needs to contain only a subset of the fields in the BigQuery schema. (:issue: `24`)
```

Comment:
no space after ``:issue:`24` ``, say more like:

``.to_gbq(...., if_exists='append')`` ...the rest of what you were saying